packages/ws-impl/package.json

```diff
@@ -3,6 +3,11 @@
     "version": "0.0.0",
     "private": true,
     "exports": {
+        "edge-light": {
+            "import": "./dist/index.node.mjs",
+            "require": "./dist/index.node.cjs",
+            "types": "./dist/types/index.browser.d.ts"
```

Comment:
Is this the only package we need to export the types for?

Reply:
I think there was one other too - I copied the `node` export, which included `types` only in these cases. I'm not sure why that is but figured I should just do the exact same for `edge-light`.